src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -89,20 +89,24 @@ internal static class TestCommandParser
         public static readonly Option<bool> BlameCrashOption = new ForwardedOption<bool>("--blame-crash", LocalizableStrings.CmdBlameCrashDescription)
             .ForwardAs("-property:VSTestBlameCrash=true");
 
-        public static readonly Argument<string> BlameCrashDumpArgument = new Argument<string>(LocalizableStrings.CrashDumpTypeArgumentName).FromAmong(new string[] { "full", "mini" });
-
         public static readonly Option<string> BlameCrashDumpOption = new ForwardedOption<string>("--blame-crash-dump-type", LocalizableStrings.CmdBlameCrashDumpTypeDescription)
-            .ForwardAsMany(o => new[] { "-property:VSTestBlameCrash=true", $"-property:VSTestBlameCrashDumpType={o}" });
+        {
+            ArgumentHelpName = LocalizableStrings.CrashDumpTypeArgumentName,
+        }
+        .FromAmong(new string[] { "full", "mini" })
+        .ForwardAsMany(o => new[] { "-property:VSTestBlameCrash=true", $"-property:VSTestBlameCrashDumpType={o}" });
 
-        public static readonly Option<string> BlameCrashAlwaysOption = new ForwardedOption<string>("--blame-crash-collect-always", LocalizableStrings.CmdBlameCrashCollectAlwaysDescription)
+        public static readonly Option<bool> BlameCrashAlwaysOption = new ForwardedOption<bool>("--blame-crash-collect-always", LocalizableStrings.CmdBlameCrashCollectAlwaysDescription)
```

Comment:
it looks like the intent of this property is to never let the user specify a `false` value. for these kinds of flag-style properties, consider setting `{ Arity = ArgumentArity.Zero }` so that a user passing in an explicit true/false value gets a parse error instead of the boolean value getting silently parsed and then discarded by the forwarding.